perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -723,6 +723,7 @@ private void serve(HttpServletRequest req, HttpServletResponse resp, boolean isG
 					result = caller.call(manager, method, des);
 					if (perunRequest != null) perunRequest.setResult(result);
 				} catch (Exception ex) {
+					log.warn("Exception {}: {}.", Long.toHexString(System.currentTimeMillis()), this);
```

Comment:
This line is not correct, the id logged is different than the id of the exception, because the id logged is the hexadecimal number of milliseconds in the time of catching the exception, while the id of the exception is the time when the exception was created.

Reply:
Done.